### ISSUE TITLE
No access token field

### ISSUE THREAD

**Issue by samwilson** [publifactory/Overland_android#16]
The 'Receiver Endpoint' popup form is supposed to have "an access token which will be sent in the HTTP Authorization header", but it doesn't seem to exist. Is there a way to enable this?

(I'm trying to add Overland support to a blogging tool: https://github.com/samwilson/twyne/pull/79 )

**Comment by samwilson** [comment]
Any ideas about this?

**Comment by gedankenstuecke** [comment]
Hey Sam, thanks for filing this and sorry for having missed it back in March! Unfortunately I don't have access to an Android phone for testing right now, but I've checked the iOS version which also only gives two different fields: 

1. the endpoint URL
2. the device ID

The description in iOS says _"Enter a URL above and the app will send its data there. Note that there is no authentication other than this URL, so make sure the URL you enter contains your API key or other token"_, which seems to reflect what the Android version here is doing? :-)

**Comment by samwilson** [comment]
That sounds like a good solution, thanks! And no worries about the delay, my project is not a fast-moving thing (although, once I get this set up, I guess I'll know at least how fast! :-P)

I think the intention is that the app send an Authorization header though, as that's also what the [example](https://github.com/OpenHumans/Overland_android/blob/380c1d5bc33b5a46a68047ccd0ae31d1ac5250a1/README.md#api) shows. Or maybe the docs are wrong?

I'd be happy to help with testing on Android, if there's anything I can do.

**Comment by gedankenstuecke** [comment]
Right, the intention seems to be to send the authorization headers, but I checked the iOS version of Overland on which we've based the Android one and there doesn't seem to be a way to add the tokens etc. either (see screenshot below). 

![IMG_9BA79F3F86D7-1](https://user-images.githubusercontent.com/674899/133390359-1aa1ff60-c707-4a0e-a497-250ad78a0f72.jpeg)

pinging @aaronpk: I wonder if that's was a planned feature that hasn't made it into the iOS version yet. And as we basically just ported the Android version it's not in that one either ;-)
